stubs/qrcode/qrcode/image/pil.pyi

```diff
@@ -1,11 +1,28 @@
-from _typeshed import Incomplete
+from typing import IO, Any, Literal
 
-import qrcode.image.base
+from PIL import Image
 
-class PilImage(qrcode.image.base.BaseImage):
-    kind: str
-    fill_color: Incomplete
-    def new_image(self, **kwargs): ...
-    def drawrect(self, row, col) -> None: ...
-    def save(self, stream, format: Incomplete | None = None, **kwargs) -> None: ...  # type: ignore[override]
-    def __getattr__(self, name): ...
+from . import base
+
+class PilImage(base.BaseImage):
+    kind: Literal["PNG"]
+    fill_color: str
+    # the new_image and get_image methods accept arbitrary keyword arguments to
+    # accommodate subclasses with additional arguments.
+    def new_image(self, *, back_color: str = "white", fill_color: str = "black", **kwargs: Any) -> Image.Image: ...
+    def get_image(self, **kwargs: Any) -> Image.Image: ...
+    def drawrect(self, row: int, col: int) -> None: ...
+    # kwargs are passed on to PIL.Image.save, which also accepts arbitrary keyword arguments.
+    def save(  # type: ignore[override]
+        self,
+        stream: IO[bytes],
```

Comment:
Considering this is forwarded to Pillow's `Image.save()`, it should take the same type:

```suggestion
        stream: str | bytes | Path | Image._Writeable,
```

(`Path` needs to be imported from `pathlib`.)

Reply:
Good catch, I forgot about these.